### ISSUE TITLE
Does python scripts acceptable?

### ISSUE THREAD

**Issue by mnwato** [Targoman/PersianWebScraper#122]
According to current scrapper, your main programming language is TypeScript. 
Is it possible to develop the crawler/scrapper by other languages such as Python?

**Comment by ziabary** [comment]
It is hard to reimplement it by python. Why do you need to reimplement it?

**Comment by mnwato** [comment]
No, I didn't mean rewriting in another language. According to the contribution requirements published in the link bellow, I was thinking about creating specific scrapper. That's why I asked.
https://oss.targoman.ir/TLPC

**Comment by ziabary** [comment]
I think you can better contribute by writing Python scripts to preprocess jsonl.gz files in order to ease training process
